backport/main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/v50/github"
+	"github.com/sethvargo/go-githubactions"
+)
+
+type Inputs struct {
+	Title                  string
+	Labels                 []*github.Label
+	RemoveDefaultReviewers bool
+}
+
+func GetInputs() Inputs {
+	var (
+		labelsStr                 = githubactions.GetInput("labelsToAdd")
+		removeDefaultReviewersStr = githubactions.GetInput("removeDefaultReviewers")
```

Comment:
I don't see `removeDefaultReviewers` or `title` defined as inputs in the `backport.yml`. Is this an issue?

Reply:
yea i'll have to remove those